python/tests/conftest.py

```diff
@@ -58,3 +74,65 @@ def resnet_model_uri(tmp_path_factory):
     model_uri = tmp_path / "resnet.pth"
     torch.save(resnet, model_uri)
     return model_uri
+
+
+@pytest.fixture
+def gcs_tmpdir() -> str:
+    """Create a temporary Google Cloud Storage (GCS) directory to test dataset.
+
+    To enable GCS test, it requires both the AWS credentials,
+    as well as `RIKAI_TEST_GCS_BUCKET` being set.
+
+    Examples
+    --------
+
+    .. code-block:: bash
+
+        $ export GOOGLE_APPLICATION_CREDENTIALS=/path/to/key.json
+        $ export RIKAI_TEST_GCS_BUCKET=gs://bucket
```

Comment:
Should bucket also contain the `gs://` part? I got confused reading line 125 where I didn't realize the `bucket` variable was prefixed by `gs://`.

Reply:
Good point, would the env var to be named as `RIKAI_TEST_GCS_PATH` be more appropriated?

Reply:
personally i'd prefer `gs://...`, it is more explicit , and this is the form that used in spark and python reader.

Reply:
Okay, can you add a check that the yielded `temp_dir` contains a `gs://` prefix then? Otherwise it's possible for someone to specify `export RIKAI_TEST_GCS_BUCKET=bucket` and end up writing to a local file `bucket/xyz123` when doing `df.write.format("rikai").save(tmpdir)`.

Reply:
I will add a check on the  input then. So it enforces user to use `gs://`